Sources/Orbit/Components/Alert.swift

```diff
@@ -242,13 +317,19 @@ struct AlertPreviews: PreviewProvider {
     }
 
     static var standalone: some View {
-        Alert(
-            "Alert with very very very very very very very very very very very long and <u>multiline</u> title",
-            description: description,
-            icon: .grid,
-            buttons: primaryAndSecondaryConfiguration
-        ) {
-            contentPlaceholder
+        VStack(spacing: .large) {
+            Alert(
+                "Alert with very very very very very very very very very very very long and <u>multiline</u> title",
+                description: description,
+                icon: .grid,
+                buttons: primaryAndSecondaryConfiguration
+            ) {
+                contentPlaceholder
+            }
+
+            Alert("Alert with very very very", icon: .grid, button: "Primary", status: .warning, isSuppressed: false)
```

Comment:
This one is probably meant to include the `.idealSize(horizontal: true)` as these are now the same in snapshots?

Reply:
Oh, this is an oversight from adding the one below - this one is the "default" inline alert and was previously named that - `Inline alert`. I'll revert the name.